elements/bulk/nuxeo-edit-documents-button.js

```diff
@@ -545,13 +584,8 @@ class NuxeoEditDocumentsButton extends mixinBehaviors([I18nBehavior, FiltersBeha
     }
     const bulkLayout = this.$$('nuxeo-layout').element;
     const value = bulkLayout.get(boundPath);
-    // XXX If we knew the data types for each property type we wouldn't need to infer it.
-    if (Array.isArray(value)) {
-      if (value.length !== 0) {
-        bulkLayout.set(boundPath, []);
-      }
-    } else {
-      bulkLayout.set(boundPath, null);
+    if (value && value.length !== 0) {
```

Comment:
I assume what we wanted here was this?
```suggestion
    if (Array.isArray(value) && value.length > 0) {
```

Reply:
➕ 1